code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3199,4 +3199,16 @@
 	strength = 9
 
 	glass_name = "fusionnaire"
-	glass_desc = "A relatively new cocktail, mostly served in the bars of NanoTrasen owned stations."
\ No newline at end of file
+	glass_desc = "A relatively new cocktail, mostly served in the bars of NanoTrasen owned stations."
+	
+/datum/reagent/ethanol/cider
+	name = "Cider"
+	id = "cider"
+	description = "A sparkling alcoholic beverage derived from apples, quite refreshing."
+	taste_description = "farmyard apples"
+	color = "#EAB300"
+	strength = 9
```

Comment:
This might be a tad strong for cider? Minor thing.

Reply:
I've weakened it down to British cider levels. Strength is now 5.

Reply:
I probably should have elaborated; for whatever reason, ethanol based reagents with a lower strength are more alcoholic. Ergo, a drink with a strength of 5 is actually more potent than one with a strength of 6.


I also just noticed that there is already a cider reagent, see attached link. If you'd care to remove, replace, or otherwise edit that version, rather than adding a second one, it would be much appreciated. Sorry for not seeing that sooner.
https://github.com/PolarisSS13/Polaris/blob/a033ac4779e6a36c3e58cacbb008c45ccc4fc985/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm#L1839

Reply:
I must be blind.

Nonetheless, at least we have bottles now. I'll edit this in a bit and leave the original description.